### ISSUE TITLE
Woo: No translate product attributes in order item name

### ISSUE THREAD

**Issue by yunusga** [VaLeXaR/wp-multilang#125]
If we comment this conditional, all works fine.
Why was this condition necessary?
https://github.com/VaLeXaR/wp-multilang/blob/f56b620586a9b53cde0cec8d8815b87ed211db48/includes/wpm-translation-functions.php#L434

**Comment by VaLeXaR** [comment]
Because attributes are taxonomies and create automatically. You should add attribute term taxonomy to config for translation.

**Comment by yunusga** [comment]
Yes, but, control over categories and attributes in manual mode is not very convenient. Because categories and attributes can be a huge number and keep track of everything, will be very difficult, which will entail errors in the plugin.
